src/transformers/models/prompt_depth_anything/__init__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
This uses the old init, but this can be updated to the new form.

Basically like here: https://github.com/huggingface/transformers/blob/main/src/transformers/models/clip/__init__.py.
You then also need to include the following in `modular_prompt_depth_anything.py`: https://github.com/huggingface/transformers/blob/ccc4a5a59b2d4134a49971915db0710e7a8c7824/src/transformers/models/clip/modeling_clip.py#L1681-L1689, and run the modular tool again to update the modeling file